lib/lab_manager/provider/vsphere.rb

```diff
@@ -285,5 +290,9 @@ def machine_present_in_drs_rule?(vs, group:, machine:, datacenter:)
       machine_short_name = machine.sub!(%r{^.*\/}, '')
       !(group.vm.find { |v| v.name == machine_short_name }).nil?
     end
+
+    def retryable_exception
```

Comment:
1. use this method/callback  in almost every retryable usuage.
2. define it as constant RETRYABLE_CALLBACK = ->(ex) { ... }